ark/phenotyping/cluster.py

```diff
@@ -0,0 +1,151 @@
+import os
+
+import numpy as np
+import pandas as pd
+
+
+def winner(sample, weights):
+    """Find the coordinates to the winning neuron in the SOM
+
+    Args:
+        sample (numpy.ndarray):
+            A row in the pixel matrix identifying information associated with one pixel
+        weights (numpy.ndarray):
```

Comment:
This description is confusing to me. I thought the weights matrix for the SOM represented the location in n-dimensional space for each of the neurons in the SOM? Isn't this matrix `[num_x * num_y * num_channels]`?

Reply:
Yes, it should be. I'll make sure to clear that up with the `cluster.py` PR off of this one.